### ISSUE TITLE
bug: component 5 has no LABEL

### ISSUE THREAD

**Issue by mercurier** [Gibberlings3/Jasteys_SoD_Tweakpack#25]
```
BEGIN @1008 /* ~More Dialogue Choices & Prevent Dead Ends~ */ DESIGNATED 5
REQUIRE_PREDICATE ((GAME_IS ~eet~) OR (GAME_IS ~bgee~ AND FILE_EXISTS_IN_GAME ~bd0103.are~))  @1000 /* ~This component is only compatible with SoD and EET.~ */
FORBID_COMPONENT ~EET_end.tp2~ ~0~ @1017
FORBID_COMPONENT ~EET_end.tp2~ ~1~ @1017
```

**Comment by Gitjas** [comment]
And it's not the only one. Thanks for reporting!

**Comment by Gitjas** [comment]
Fixed in v10.2
